### PR TITLE
Add generic content action callback in ux_sync lib.

### PR DESCRIPTION
**Pull request by agrojean-ledger**

### CODE REVIEW

File: lib_ux_sync/src/ux_sync.c [hunk 1]
```diff
@@ -1,10 +1,24 @@
...
+void generic_content_action_callback(int token, uint8_t index, int page)
+{
+    UNUSED(index);
+    UNUSED(page);
+    if (token == FIRST_USER_TOKEN) {
+        g_ret = UX_SYNC_RET_APPROVED;
+    }
+    else if (token == FIRST_USER_TOKEN + 1) {
+        g_ret = UX_SYNC_RET_REJECTED;
+    }
```
Comment (fbeutin-ledger) [inline lib_ux_sync/src/ux_sync.c:18]: g_ret might end up unmodified, is this expected ?
If so write a comment to explain the case where it happens

**Comment by tdejoigny-ledger** [comment]
we should target master branch I guess

File: lib_ux_sync/src/ux_sync.c (hunk 1 above)
Comment (agrojean-ledger) [reply lib_ux_sync/src/ux_sync.c:18]: Not sure we want this to happen... I might end up just using an "else" :thinking:

File: lib_ux_sync/src/ux_sync.c [hunk 2]
```diff
@@ -1,10 +1,24 @@
 #ifdef HAVE_NBGL
 
 #include "ux_sync.h"
+#include "os_helpers.h"
 
 static ux_sync_ret_t g_ret;
 static bool          g_ended;
 
+void generic_content_action_callback(int token, uint8_t index, int page)
```
[the remark this answers is not in the document]
Comment (agrojean-ledger) [reply lib_ux_sync/src/ux_sync.c:9]: Done.

[the remark this answers is not in the document]
Comment (agrojean-ledger) [reply lib_ux_sync/src/ux_sync.c:9]: @fbeutin-ledger new PR on master with changes according to your request : https://github.com/LedgerHQ/ledger-secure-sdk/pull/705

File: lib_ux_sync/src/ux_sync.c (hunk 1 above)
Comment (agrojean-ledger) [reply lib_ux_sync/src/ux_sync.c:18]: @fbeutin-ledger new PR on master with changes according to your request : https://github.com/LedgerHQ/ledger-secure-sdk/pull/705

**Comment by agrojean-ledger** [comment]
@tdejoigny-ledger new PR on master https://github.com/LedgerHQ/ledger-secure-sdk/pull/705

**Pull request closed without merging**